EXT_BeamPhysics.md

```diff
@@ -0,0 +1,278 @@
+Extension to the openPMD Standard for Describing Particle Beams and X-rays
+==========================================
+
+Version 2.0.0
+
+Overview
+------------
```

Comment:
@ax3l : Sorry I don't understand this.

Reply:
I think that @ax3l meant that you should use
```
Overview
--------
```
instead of
```
Overview
------------
```
(i.e. The number of dashes should equal the number of characters in "Overview")

Reply:
@ax3l Ah! Now I understand. Thanks.

Reply:
OK changed in the coming revision. Note: Markdown does not *require* this.